src/main/java/org/broadinstitute/dsm/pubsub/KitTrackerPubSubPublisher.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.dsm.pubsub;
+
+import com.google.api.core.ApiFuture;
+import com.google.api.core.ApiFutureCallback;
+import com.google.api.core.ApiFutures;
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.common.util.concurrent.MoreExecutors;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.ProjectTopicName;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.TopicName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+
+
+public class KitTrackerPubSubPublisher {
+
+    private final Logger logger = LoggerFactory.getLogger(KitTrackerPubSubPublisher.class);
+    public void publishWithErrorHandlerExample(String projectId, String topicId, String message)
```

Comment:
I'm guessing this was copy-pasted from somewhere?  How about renaming to `publishMessage()`?